src/moonstone-samples/lib/AccessibilitySample.js

```diff
@@ -23,32 +24,56 @@ module.exports = kind({
 	labelText : 'Label',
 	hintText : 'Hint',
 	components: [
-		{kind: Header, name: 'header', content: 'Accessibility', type: 'small', components: [
+		{kind: Header, name: 'header', content: 'Accessibility', titleBelow: 'Unusual case sample for Accessibility', type: 'small', components: [
 			{name: 'labelButton', kind: Button, small: true, minWidth: false, content: 'Set Label', ontap: 'labelButtonTapped'},
 			{name: 'hintButton', kind: Button, small: true, minWidth: false, content: 'Set Hint', ontap: 'hintButtonTapped'},
-			{name: 'toggle', kind: ToggleButton, small: true, toggleOnLabel: 'disabled is on', toggleOffLabel: 'disabled is off', ontap: 'disabledTapped'},
-			{name: 'reload', kind: Button, small: true, content: 'Reload', ontap: 'reload'}
+			{name: 'toggle', kind: ToggleButton, small: true, toggleOnLabel: 'disabled is on', toggleOffLabel: 'disabled is off', ontap: 'disabledTapped'}
 		]},
 		{name: 'container', kind: Scroller, fit: true, components: [
 			{classes: 'moon-1v'},
-			{kind: Divider, content: 'Icons: '},
-			{kind: Icon, icon: 'play', small: false, ontap: 'buttonTapped', accessibilityLabel: 'play'},
-			{kind: Icon, src: '@../assets/icon-list.png', ontap: 'buttonTapped', accessibilityLabel: 'icon List'},
-			{kind: Item, classes: 'moon-hspacing', ontap: 'buttonTapped', accessibilityLabel: 'selectable icon List', components: [
-				{content: 'Selectable Item', accessibilityDisabled: true},
+			{kind: Control, tag: 'br'},
+			{kind: Divider, content: 'Case 1: Non spotlight'},
+			{kind: Icon, icon: 'play', small: false, ontap: 'buttonTapped'},
+			{kind: Icon, src: '@../assets/icon-list.png', ontap: 'buttonTapped'},
+			{classes: 'moon-1v'},
+			{kind: Control, tag: 'br'},
+			{kind: Divider, content: 'Case 2: User defined'},
+			{kind: Item, classes: 'moon-hspacing', ontap: 'buttonTapped', components: [
+				{content: 'Content and Icon'},
 				{kind: Icon, src: '@../assets/icon-list.png', ontap: 'buttonTapped'}
 			]},
-			{kind: Control, tag: 'br'},
+			{name: 'checkItem', kind: Item, classes: 'moon-hspacing', ontap: 'buttonTapped', components: [
```

Comment:
If the goal here is to represent how to make a custom kind that is accessible, it should be defined as such separate from the sample. That will give us the opportunity to show the accessibility-specific members (e.g. `ariaObservers`, `accessibilityRole`, etc).